### DISCUSSION TITLE
Model ID and Model Code of the new HERO

### DISCUSSION THREAD

**Discussion by rhoenschrat** [gopro/OpenGoPro#613] [Q&A]
Hi
Can anyone provide the Model ID and Model Code for the new GoPro HERO (4K)?
I did not see it listed on the BLE Specs pgae.

**Comment by KonradIT** [comment]
Likely not supported by OpenGoPro, but:

Model ID: 66 

Code: H24.03

**Reply by rhoenschrat** [reply]
Thanks a lot.
